pyproject.toml

```diff
@@ -1,13 +1,35 @@
 [tool.poetry]
-name = "tap-postgres"
-version = "0.0.1"
-description = "`tap-postgres` is a Singer tap for Postgres, built with the Meltano SDK for Singer Taps."
-authors = ["AutoIDM"]
+name = "meltanolabs-tap-postgres"
+version = "0.0.0"
+description = "`tap-postgres` is a Singer tap for Postgres, built with the Meltano SDK for Singer Targets."
+authors = ["Meltano Team and Contributors"]
+maintainers = ["Meltano Team and Contributors"]
```

Comment:
```suggestion
authors = ["Meltano Team and Contributors <hello@meltano.com>"]
maintainers = ["Meltano Team and Contributors <hello@meltano.com>"]
```

Reply:
Is there a standard for this stuff out there somewhere? I'm just guessing based on `target-postgres` here